plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -111,7 +111,10 @@ export class EventPipelineRunner {
     }
 
     async runEventPipelineSteps(event: PluginEvent): Promise<EventPipelineResult> {
-        if (this.hub.poeEmbraceJoinForTeams?.(event.team_id)) {
+        if (
+            this.hub.poeEmbraceJoinForTeams?.(event.team_id) ||
+            (event.team_id <= this.hub.POE_WRITES_ENABLED_MAX_TEAM_ID && !this.hub.poeWritesExcludeTeams(event.team_id))
```

Comment:
Organization ID would be preferable for our purposes here in `poeWritesExcludeTeams` but the full `Team` isn't in scope within this function.

I didn't want to do a refactor just for this check, and this will only be around for a short while (at least I think so, … 🤞) and the exclusion set isn't _that_ big, so I think this is fine for what we need.